CHANGELOG.md

```diff
@@ -2,3 +2,6 @@
 
 ## 1.0.0
 - Plugin released 🎉
+
```

Comment:
Sorry, one last change please remove all changes done in this file

Reply:
done,